src/wdio/services/compare/TerraCompare.js

```diff
@@ -0,0 +1,55 @@
+import BaseCompare from 'wdio-visual-regression-service/lib/methods/BaseCompare';
+import resemble from 'node-resemble-js';
+import get from 'lodash.get';
+
+export default class TerraCompare extends BaseCompare {
+  constructor(config) {
+    super();
+    this.screenshotProcessor = config.screenshotProcessor;
+    this.failInCIOnMissingReferenceShots = config.failInCIOnMissingReferenceShots;
+  }
+
+  async processScreenshot(context, base64Screenshot) {
+    const referenceScreenshotData = await this.screenshotProcessor.getReferenceScreenshotData(context);
+    const latestScreenshotData = Buffer.from(base64Screenshot, 'base64');
+    await this.screenshotProcessor.saveLatestScreenshotData(context, latestScreenshotData);
+    if (referenceScreenshotData) {
+      const comparisonData = await TerraCompare.compareImages(referenceScreenshotData, latestScreenshotData);
+      const { isSameDimensions } = comparisonData;
+      const misMatchPercentage = Number(comparisonData.misMatchPercentage);
+      const misMatchTolerance = get(context, 'options.misMatchTolerance', 0.01);
```

Comment:
I feel like our misMatchTolerance should be increased. It seems we've run into a lot of diffs that are related to anti-aliasing text or other subpixel diffs.